packages/arb-token-bridge-ui/src/util/orbit/orbitMainnets.json

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
Please double check that I didn't miss an Orbit chain while migrating, because I was resolving some conflicts too.

Also, pls compare old data against the new.